apps/v3_api/test/alerts_test.exs

```diff
@@ -0,0 +1,17 @@
+defmodule V3Api.AlertsTest do
+  use ExUnit.Case
+
+  describe "all" do
+    test "hits /alerts" do
+      bypass = Bypass.open()
+
+      Bypass.expect(bypass, fn conn ->
+        assert conn.request_path == "/alerts/"
+        conn = Plug.Conn.fetch_query_params(conn)
+        Plug.Conn.resp(conn, 200, ~s({"data": []}))
+      end)
+
+      assert %JsonApi{} = V3Api.Alerts.all()
+    end
+  end
```

Comment:
What are you trying to cover here? This goes to the /alerts page of the website, it's not equivalent to fetching /alerts in the V3 API.